sklearn/calibration.py

```diff
@@ -942,6 +1107,67 @@ def predict(self, T):
         return expit(-(self.a_ * T + self.b_))
 
 
+class _TemperatureScaling(RegressorMixin, BaseEstimator):
+    """Temperature scaling model.
+
+    Attributes
+    ----------
+    T_initial_ : float
+        The initial temperature for optimization.
+
+    T_ : float
+        The optimized temperature.
```

Comment:
Maybe we can just expose inverse temperature (aka the beta values), both initial and fitted directly instead of keeping them private attributes. The I would drop the `T_initial` parameter and `T_` attribute.